hathor/manager.py

```diff
@@ -264,6 +265,8 @@ def start(self) -> None:
 
         # Disable get transaction lock when initializing components
         self.tx_storage.disable_lock()
+        # Open scope for initialization.
+        self.tx_storage.set_allow_scope(TxAllowScope.VALID | TxAllowScope.PARTIAL | TxAllowScope.INVALID)
```

Comment:
Should we use `set_allow_scope()` here? In its docstring, there's: `This method should not normally be used directly, use one of the allow_*_scope methods instead`

Reply:
This is a special occasion because we need to access all transactions during initialization. So, we allow full access to the DAG and restore normality after the full node is initialized.

Reply:
I'll merge it and change it later if you disagree.